sakura_core/dlg/CDlgSetCharSet.h

```diff
@@ -18,7 +18,7 @@
 
 #include "dlg/CDialog.h"
 
-enum ECodeType;
+enum ECodeType : int;
```

Comment:
主題「前方宣言しないと怒られる」の対策は以下で解決できそうに思います。

```suggestion
#include "charset/charset.h"
```

主題以外の修正が多いですが、「ダメ」でない気はしてます。